pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -19,6 +24,14 @@ type nowFunc func() metav1.Time
 const (
 	// CatalogSourceLabelKey is the key for a label containing a CatalogSource name.
 	CatalogSourceLabelKey string = "olm.catalogSource"
+	// RedHatCatalogAnnotationKey is the key of an annotation in default catalogsources
+	RedHatCatalogAnnotationKey string = "operatorframework.io/managed-by"
+	// RedHatCatalogAnnotationValue is the value of an annotation in default catalogsources
+	RedHatCatalogAnnotationValue string = "marketplace-operator"
+	// RedHatCatalogPriorityClassKey is the key of an annotation in default catalogsources
+	RedHatCatalogPriorityClassKey string = "operatorframework.io/priorityclass"
```

Comment:
Do these constant names need "RedHat" in the name? Can we choose more upstream-friendly names?

Reply:
Yeah agreed - I think prefixing variables with ocp/redhat/etc. makes sense if it's a downstream-specific API, but priority classes are an upstream concept: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority.